releases/README.md

```diff
@@ -0,0 +1,86 @@
+# Reproducible builds
+
+The bitbox02 firmware [releases](https://github.com/digitalbitbox/bitbox02-firmware/releases/) are
+tagged: `firmware/vX.Y.Z` for the Multi edition, and `firmware-btc-only/vX.Y.Z` for the Bitcoin-only
+edition. The binaries are built from those tags in a reproducible manner, based on Docker.
+
+*Note*: it is possible to reproduce the binaries without Docker, the only really relevant dependency
+is the ARM GCC compiler referenced in the [Dockerfile](../Dockerfile). The instructions below use
+Docker however as it makes it a easier to get started.
+
+## Contribute your signature!
+
+We kindly ask you to independently build the firmware binaries we released, and verify that you get
+the same result. Please open a PR to add your signed message confirming this. Signing a confirmation
+like this *only* confirms that the released binary matches the build from the specified tag. It
+*does not* endorse the contents or quality of the binary itself.
+
+### How to reproduce:
+
+Clone this repo:
+
+```sh
+git clone --recurse-submodules https://github.com/digitalbitbox/bitbox02-firmware
+```
+
+Checkout the version you want to reproduce, for example version 4.1.0:
+
+```sh
+git checkout firmware/v4.1.0
+# or a Bitcoin-only firmware: git checkout firmware-btc-only/v4.1.0
+```
+
+Clean any potential previous build results and caches:
+
+```sh
+make clean
+```
+
+Build the Docker image (this can take a while):
+
+```sh
+docker build --pull --force-rm --no-cache -t bitbox02-firmware .
+```
+
+Build the firmware:
+
+```sh
+docker run -it --rm --volume `pwd`:/bb02 bitbox02-firmware make firmware
+# or for Bitcon-only
+docker run -it --rm --volume `pwd`:/bb02 bitbox02-firmware make firmware-btc
+```
+
+There binary will be at `build/bin/firmware.bin` (resp. `build/bin/firmware-btc.bin`).
+
+(TODO)
+
+**Comparing to the official release**
+
+Per release there are two files: the raw binary `.bin`, and a signed version `.signed.bin`. The
+signed version is the same as the raw binary, but prefixed with some signature data which the
+BitBox02 verifies before running the firmware.
+
+(TODO: describe how to compare, how provide a script to derive the "firmware hash" as displayed by
+the bootloader.)
+
+**Contributing your signature**
+
+If you verified successfully that the released binary and the binary you built match, please copy
+[this message template](template-assert.txt) to `YourName-firmware-vX.Y.Z-assert.txt` and sign it
+using:
+
+(TODO)
+
+## What is the purpose of this?
+
+The purpose is twofold:
+
+- It is much more technically involved to actually build the binaries from source, than it is to
+  verify signatures using `gpg` on the command line or in a GUI. This gives users who are willing
+  and eager to verify the option to verify our releases even if they are unable to build the
+  binaries themselves.
+- The firmware binaries are only truly reproducible for a limited amount of time after the
```

Comment:
Only a problem with the instructions. They are truly reproducible forever if you specify the docker container with a numbered tag. Reproducibly rebuilding the docker container on the other hand is not possible forever, but I think that is out of scope for reproducible builds.

Reply:
As the paragraph says, in the future Docker might not even be compatible with older images, etc. etc.

I added `practically` as a qualifier, otherwise the rabbit hole goes deep :smile: